### DISCUSSION TITLE
Is it possible for changes in templates to be reflected in notes without reinserting the template? Like a class.

### DISCUSSION THREAD

**Discussion by hollowhemlock** [SilentVoid13/Templater#523] [Help]
A template I have is used in multiple files. If I make a change to the template I'd like it to be reflected in all these files without having to reinsert it. Is that possible?

**Comment by shabegom** [comment]
The best approach would be to keep this template in its own note and then use `tp.file.include()` to add it to any note where it would be used.

**Reply by hollowhemlock** [reply]
Hey,

Thanks for pointing me to that function. I got it to work in the normal non-dynamic situation
`<% tp.file.include("[[area template]]") %>`

I couldn't get it to work dynamically.
`<%+ tp.file.include("[[area template]]") %>`

**Reply by shabegom** [reply]
Yeah, that isn't going to work. I would use a transclusion ![[]] if I want a view that can be continuously updated...

**Reply by hollowhemlock** [reply]
In my case I'd like to use the title of the unique file as a variable to influence tasks queries and dataview queries that I'd like to be stored in a template file.

**Comment by hollowhemlock** [comment]
For those interested this can be done dynamically using the dv.view call in the dataview plugin. https://blacksmithgu.github.io/obsidian-dataview/api/code-reference/
